data/weapons.txt

```diff
@@ -858,14 +858,14 @@ outfit "Typhoon Launcher"
 		"hit effect" "large explosion"
 		"die effect" "tiny explosion"
 		"inaccuracy" 5
-		"velocity" 5
+		"velocity" 3
 		"lifetime" 600
 		"reload" 100
 		"firing energy" 4
 		"firing heat" 50
-		"acceleration" .5
+		"acceleration" .4
 		"drag" .1
-		"turn" 6
+		"turn" 1
```

Comment:
Should still probably be better than the Torpedo, maybe 1.75 to 2.75 range.

Reply:
I dunno, the description says "However, they are also much slower, because they are intended mostly for use against large and slow-moving capital ships."

Reply:
I'd say around 1.25-1.4 turn would be fine to make torpedoes at least decent against faster capital ships and slower regular ships